src/sass/components/_anchor-preview.scss

```diff
@@ -0,0 +1,24 @@
+.medium-editor-anchor-preview {
+    font-family: $font-sans-serif;
+    font-size: 16px;
```

Comment:
in web browsers, the `font-size` is equal to this value. Can this be a `font-size: inherit` instead in the interest of making this an easy drop-in solution on the CSS side?